### PR TITLE
Fix broken AttentionMechanism docstrings

### PR DESCRIPTION
**Pull request by jagthebeetle**
Currently, for all AttentionMechanisms, the `memory_sequence_length` arg does not get formatted correctly in the docs, making it easy to miss. 

I believe this is because the `(optional)` modifier is not preceded by a colon, as in the other (working) args.

### CODE REVIEW

File: tensorflow/contrib/seq2seq/python/ops/attention_wrapper.py [hunk 1]
```diff
@@ -152,7 +152,7 @@ def __init__(self,
       probability_fn: A `callable`.  Converts the score and previous alignments
         to probabilities. Its signature should be:
         `probabilities = probability_fn(score, previous_alignments)`.
-      memory_sequence_length (optional): Sequence lengths for the batch entries
+      memory_sequence_length: (optional): Sequence lengths for the batch entries
```
Comment (frankchn) [inline tensorflow/contrib/seq2seq/python/ops/attention_wrapper.py]: Can you remove the `:` after `(optional)` as well?

File: tensorflow/contrib/seq2seq/python/ops/attention_wrapper.py [hunk 2]
```diff
@@ -272,7 +272,7 @@ def __init__(self,
       num_units: The depth of the attention mechanism.
       memory: The memory to query; usually the output of an RNN encoder.  This
         tensor should be shaped `[batch_size, max_time, ...]`.
-      memory_sequence_length (optional): Sequence lengths for the batch entries
+      memory_sequence_length: (optional): Sequence lengths for the batch entries
```
Comment (frankchn) [inline tensorflow/contrib/seq2seq/python/ops/attention_wrapper.py]: ditto

File: tensorflow/contrib/seq2seq/python/ops/attention_wrapper.py [hunk 3]
```diff
@@ -394,7 +394,7 @@ def __init__(self,
       num_units: The depth of the query mechanism.
       memory: The memory to query; usually the output of an RNN encoder.  This
         tensor should be shaped `[batch_size, max_time, ...]`.
-      memory_sequence_length (optional): Sequence lengths for the batch entries
+      memory_sequence_length: (optional): Sequence lengths for the batch entries
```
Comment (frankchn) [inline tensorflow/contrib/seq2seq/python/ops/attention_wrapper.py]: ditto

**Review by frankchn — CHANGES_REQUESTED** [review]

**Review by jagthebeetle — COMMENTED** [review]
I found further instances of colons after `(optional)`, see next commit.

**Review by frankchn — APPROVED** [review]

**Comment by frankchn** [comment]
Jenkins, test this please.

**Comment by frankchn** [comment]
Jenkins, test this please.

**Comment by frankchn** [comment]
@jagthebeetle Can you rebase to the latest master and apply your commit on top of that? Github is showing thousands of commits right now. I think it is better to create a new PR for this rather than reusing this.

**Pull request closed without merging**